docs/sources/configure-client/go-span-profiles.md

```diff
@@ -7,38 +7,52 @@ weight: 100
 
 # Span profiles with Traces to profiles for Go
 
+Span Profiles represents a major shift in profiling methodology, enabling deeper analysis of both tracing and profiling data.
+Traditional continuous profiling provides a system-wide view over fixed intervals.
```

Comment:
Nit: system-wide or application-wide

Reply:
Which one should it be? Application-wide?

Reply:
Now that I've thought a bit more, I'm not sure if we need to clarify: I can interpret "system-wide" in a broader sense that also covers the "application-wide" case. Sorry, I didn't mean to confuse you :)

In Pyroscope, we have a strong concept of a "service" and encourage users to adhere to it. So if I had to choose, I'd pick "application-wide."

Reply:
Okay! If it is clearer to say application-wide I can change it. Although, your comment makes me wonder if I should say "system-wide and appilcation-wide"...

Reply:
```suggestion
Traditional continuous profiling provides an application-wide view over fixed intervals.
```